.github/workflows/main.yml

```diff
@@ -0,0 +1,38 @@
+name: CI
+
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+    branches: [ b2.5, b3.0, master]
+  pull_request:
+    branches: [ b2.5, b3.0, master, SPARKC-651 ]
+
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        scala:  [2.11.12]
+        db-version: [6.8.9, 3.11.9, 4.0-beta4]
```

Comment:
Can this be pointing to the latest in those series? `6.8.13`, `3.11.10`, `4.0-RC1`?